### ISSUE TITLE
Carthage: Dependency "EventSource" has no shared framework schemes

### ISSUE THREAD

**Issue by daniel-andersen** [inaka/EventSource#121]
I get this error when installing from Carthage:

```
*** Skipped building EventSource due to the error:
Dependency "EventSource" has no shared framework schemes
    
If you believe this to be an error, please file an issue with the maintainers at https://github.com/inaka/EventSource/issues/new
```

Perhaps this SO is helpful?

https://stackoverflow.com/questions/35054788/carthage-no-shared-framework-schemes-for-ios-platform-for-my-own-framework

**Comment by andresinaka** [comment]
I have to take a look. In the meantime you can just grab the files and drop them in your project or you can create a PR ❤️

**Comment by daniel-andersen** [comment]
The reason for the above error is a missing framework target. Might it be that you forgot to add the scheme to git?

I added a framework target and created a pull request. It now compiles with:
$ carthage build --no-skip-current

**Comment by andresinaka** [comment]
Thank you! Fixed #122.
